bodybank/character_generator/templates/list.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+
+{% block title %}My egos{% endblock %}
+
+{% block content %}
+  <h1>My egos</h1>
+  <div class="module">
+    {% for ego in object_list %}
```

Comment:
See comment below, for example to me it's not obvious that class based views pass on an object_list to the template, it's not transparent. With a function based view you're very explicit with what you pass through to the template via the context dictionary you pass into the render() function. So to me function based views are more explicit than class based views (Zen of Python!). Then there is the multiple levels of inheritance and hence the need for resources like this: https://ccbv.co.uk
I am not using them anymore :)
But (!) it's really good you took knowledge of them!

Reply:
I really like this excerpt: The fundamental nature of a view is that it is a function (or callable) that takes a request and returns a response.

We can switch over to function based views. They were more straightforward to work with.